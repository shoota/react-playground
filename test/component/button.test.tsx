import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../../src/components/button'

it('button', () => {
  const b = renderer.create(<Button>Example</Button>).toJSON()
  expect(b).toMatchSnapshot()
})
