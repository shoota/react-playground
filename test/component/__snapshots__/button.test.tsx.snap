// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 1`] = `
<button
  className="sc-bdVaJa jcogRD"
>
  Example
</button>
`;
